src/components/overview/ExperimentsSummary.js

```diff
@@ -0,0 +1,178 @@
+import React, {Component} from "react";
+import {connect} from "react-redux";
+import PropTypes from "prop-types";
+import {Col, Row, Spin, Statistic, Typography} from "antd";
+import CustomPieChart from "./CustomPieChart";
+import { setAutoQueryPageTransition } from "../../modules/explorer/actions";
+import {
+    overviewSummaryPropTypesShape,
+} from "../../propTypes";
+import {mapNameValueFields} from "../../utils/mapNameValueFields";
+
+const mapStateToProps = state => ({
+    overviewSummary: state.overviewSummary
+});
+
+const actionCreators = {
+    setAutoQueryPageTransition
+};
+
+class ExperimentsSummary2 extends Component {
```

Comment:
why does this have a 2 appended to the class name?

Reply:
Fixed, thanks for spotting this